src/template.html

```diff
@@ -7,7 +7,11 @@
 </head>
 <body>
     <div>
-        <ul></ul>
+        <p>Today's To Do</p>
+        <span class="reload-icon"></span>
+    </div>
+    <div class="todo-container">
+        <ul class="todo-list list-unstyled"></ul>
```

Comment:
- [ ] Kindly add the input field and please try to style the app to match the provided template 💯😄 

![image](https://user-images.githubusercontent.com/50721479/207412937-e2b6326a-40be-4358-92dd-9705c6b2ea43.png)